dev/create_vm.json

```diff
@@ -34,6 +34,7 @@
     {
       "default": {
         "type": "dynamic",
+        "ip": "10.244.16.18",
```

Comment:
Hmm, I hate having this in the public code since that's a potential security vulnerability.